sympy/polys/matrices/sdm.py

```diff
@@ -193,6 +614,31 @@ def hstack(A, *B):
         return A.new(Anew, (rows, cols), A.domain)
 
     def charpoly(A):
+        """
+        Returns the coefficients of the characteristic polynomial
+        of the :py:class:`~.SDM` matrix. These elements will be domain elements.
+        The domain of the elements will be same as domain of the :py:class:`~.SDM`.
+
+        Examples
+        ========
+
+        >>> from sympy import QQ
+        >>> from sympy.polys.matrices.sdm import SDM
+        >>> A = SDM({0:{0:QQ(1), 1:QQ(2)}, 1:{0:QQ(3), 1:QQ(4)}}, (2, 2), QQ)
+        >>> A.charpoly()
+        [1, -5, -2]
+
+        We can create a polynomial using the
+        coefficients using :py:class:`~.Poly`
+
+        >>> from sympy import Symbol
+        >>> from sympy.polys import Poly
+        >>> x = Symbol('x')
+        >>> p = Poly(A.charpoly(), x, domain=A.domain)
+        >>> p
+        Poly(x**2 - 5*x - 2, x, domain='QQ')
```

Comment:
This is very verbose. You could just combine the Symbol and Poly imports with QQ above and then this is just one line:
```
>>> Poly(A.charpoly(), Symbol('x'), domain=A.domain)
```

Reply:
Ok